test/network_interface_test.cpp

```diff
@@ -18,9 +18,9 @@ TEST(TCPInterface, testReadBEDouble)
 
 TEST(TCPInterface, testReadBEInt)
 {
-    uint32_t in = 0x499602D3; //1234567891 as  hex Int
-    uint8_t bytes[4] = { 0x49, 0x96, 0x02, 0xD3};
-    ASSERT_EQ(1234567891, AS::Network::read_be<uint32_t>(bytes, 4, 0));
+    uint32_t in = 0x34FB5E38; //1234567891 as  hex Int
+    uint8_t bytes[4] = { 0x34, 0xFB, 0x5E, 0x38};
+    ASSERT_EQ(888888888, AS::Network::read_be<uint32_t>(bytes, 4, 0));
```

Comment:
You changed the test value but not the comment that describes the value. Not sure which one you meant to change but they should be consistent (probably better to make them consistent between BE and LE tests too).

Reply:
I did notice this one and I will change that. I was seeing odd behavior and wanted to make sure it was consistent between 2 different integers.